### PR TITLE
work through kubediff CLI 

### PR DESCRIPTION
**Pull request by zapman449**
args to take --namespace and --context arguments like kubectl

### CODE REVIEW

File: kubediff [hunk 1]
```diff
@@ -31,7 +33,7 @@ supply the kubeconfig for the appropriate environment.""")
   if options.json:
     printer = JSONPrinter()
 
-  failed = check_files(args, printer, options.kubeconfig)
+  failed = check_files(args, printer, options.kubeconfig, options.context, options.namespace)
```
Comment (tomwilkie) [inline kubediff]: It might be easier here to just pass through the whole options object.

Comment (zapman449) [inline kubediff]: If that would be your preference, that works.

I may switch the whole thing over to argparse from optparse in the process...

File: kubedifflib/_kube.py [hunk 2]
```diff
@@ -46,16 +46,24 @@ def from_dict(cls, data):
...
+        fetches data from the default context's namespace.
     :return: A dict of data for this Kubernetes object.
     """
     args = ["--namespace=%s" % self.namespace, "-o=yaml"]
     if kubeconfig is not None:
       args.append("--kubeconfig=%s" % kubeconfig)
+    if context is not None:
+      args.append("--context=%s" % context)
+    if namespace is not None:
+      args.append("--namespace=%s" % namespace)
```
Comment (bboreham) [inline kubedifflib/_kube.py]: It would be better to pre-empt the `--namespace` arg on line 60

**Comment by bboreham** [comment]
Note #46 which is similar.

**Comment by bboreham** [comment]
A `--namespace` option was added in #51; I'm going to close this PR since it will need major surgery.
Thanks for submitting!

**Pull request closed without merging**